website/src/parsers/js/transformers/tslint/index.js

```diff
@@ -12,13 +12,26 @@ export default {
   defaultParserID: 'typescript',
 
   loadTransformer(callback) {
-    require(['tslint/lib/index'], tslint => callback({tslint}));
+    require([
```

Comment:
Were there any other transformers that could benefit from being written in TypeScript?